contracts/ERC20Safe.sol

```diff
@@ -13,9 +13,6 @@ import "@openzeppelin/contracts/token/ERC20/ERC20Burnable.sol";
 contract ERC20Safe {
     using SafeMath for uint256;
 
-    // ERC20 contract => amount of tokens owned by Safe
-    mapping(address => uint256) public _balances;
-
     // ERC20 contract => amount of tokens burned by Safe
     mapping(address => uint256) public _burnedTokens;
```

Comment:
I think this can be removed as well. Thoughts?

Reply:
we don't track the number of burned tokens within the ERC20 and ERC721 token standard, they simply delete from balances